src/__tests__/RemoteExplodedEpub.test.ts

```diff
@@ -80,6 +80,10 @@ describe('Moby EPUB 3 Exploded', () => {
   it('extracts correct metadata', async () => {
     const manifest = await getManifest();
 
+    expect(manifest.metadata.presentation?.layout).toBe(
+      mobyEpub3Manifest.metadata.presentation.layout
+    );
+
```

Comment:
We should add a test somewhere for the non default (`fixed`) case

Reply:
We can switch this test to check for `fixed` then, since `reflowable` will not have this property.